### PR TITLE
Quiz result page

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created quiz result page.

### CODE REVIEW

**Review by VedanthB — APPROVED** [review]
Good work ananya,
Everything looks good to go. 🚀

File: index.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.search__bar__wrapper {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search__bar {
+  margin: 0 auto;
```
Comment (ritikaawasthy) [inline index.css:11]: you could just use auto here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    <header class="w-full h-20 shadow bg-slate-900 fixed z-50">
+      <div class="flex justify-around align-items-center h-full w-full">
+        <!-- header-left (logo, nav-links) -->
+        <div class="flex justify-center align-items-center">
+          <!-- Logo -->
+          <a href="/index.html">
+            <h5
+              class="h5 cursor-pointer mr-10 text-cyan-300 logo text-small text-italic inline-flex justify-center align-items-center"
+            >
+              <!-- <img src="" alt="logo" class="w-14 h-14" /> -->
```
Comment (ritikaawasthy) [inline index.html:37]: remove comments if you're not planning to use them

**Review by ritikaawasthy — APPROVED** [review]
Do check the comments, otherwise looks great

**Pull request merged**